docs/content/surfacers/overview.md

```diff
@@ -112,3 +112,77 @@ surfacer {
 }
 ```
 (Source: https://github.com/google/cloudprober/blob/master/surfacers/proto/config.proto)
+
+## Additional metrics
+#### Adding your own labels
```

Comment:
Thinking more about it, I think it's better to add it as an a new page: 

`how-to/additional-labels.md` with the following header:

```
---
menu:
    main:
        parent: "How-Tos"
        weight: 26
title: "Additional Labels"
date: 2021-06-28T17:24:32-07:00
---
```